toolchains/syno-armada38x-5.2/Makefile

```diff
@@ -0,0 +1,16 @@
+TC_NAME = syno-$(TC_ARCH)
+
+TC_ARCH = armada38x
+TC_VERS = 5.2
+TC_FIRMWARE = 5.2-5644
+
+TC_DIST_NAME = armada38x-gcc493_glibc220_hard-GPL.txz
+
+TC_EXT = txz
+TC_DIST_SITE = http://sourceforge.net/projects/dsgpl/files/DSM%205.2%20Tool%20Chains/Marvell%20Armada%2038x%20Linux%203.10.35
+
+TC_BASE_DIR = arm-marvell-linux-gnueabi
+TC_PREFIX = arm-marvell-linux-gnueabi
+TC_TARGET = arm-marvell-linux-gnueabi
+
+include ../../mk/spksrc.tc.mk
```

Comment:
I believe we'll need a `FIX_TARGET` override in here as well, although I could be wrong.

When not running as `root`, do a `make clean`. If you get a permission denied error, you can fix permissions as follows:

```
FIX_TARGET = myFix

include ../../mk/spksrc.tc.mk

.PHONY: myFix
myFix:
	chmod -R u+w $(WORK_DIR)
```